src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/products/ProductSyncIT.java

```diff
@@ -161,7 +161,7 @@ private void errorCallback(@Nonnull final String errorMessage, @Nullable final T
     }
 
     private List<UpdateAction<Product>> beforeUpdateCallback(@Nonnull final List<UpdateAction<Product>> updateActions,
-                                                             @Nonnull final ProductDraft productDraft,
+                                                             @Nonnull final ProductDraft newProductDraft,
```

Comment:
isn't `newProductDraft` redundant?

Reply:
We decided to use `oldProduct` and `newProductDraft` as convention across whole code for clarity: https://github.com/commercetools/commercetools-sync-java/pull/175#discussion_r148768256  #176

Reply:
my point was that the variable is not used, but i just missed missed that it is used as a function reference. so, everything is fine here